pom.xml

```diff
@@ -274,15 +274,20 @@
                 <version>0.6</version>
             </dependency>
             <dependency>
-                <groupId>asm</groupId>
+                <groupId>org.ow2.asm</groupId>
                 <artifactId>asm-all</artifactId>
-                <version>3.3.1</version>
+                <version>5.0.4</version>
             </dependency>
             <dependency>
                 <groupId>antlr</groupId>
                 <artifactId>antlr</artifactId>
                 <version>${antlr.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.ibm.icu</groupId>
```

Comment:
What's the requirement for using the icu MessageBundle instead of the builtin one? I'd prefer to have this in a separate pull requests if there are good reasons for using it. It's a 10MB dependency that maybe not everyone wants to depend on.